lib/modules/manager/github-actions/extract.ts

```diff
@@ -86,6 +88,40 @@ function extractContainer(container: unknown): PackageDependency | undefined {
   return undefined;
 }
 
+const runnerVersionRegex = regEx(
+  /^\s*(?<depName>[\d\w]+)-(?<currentValue>[^\s]+)/
```

Comment:
```suggestion
  /^\s*(?<depName>[a-z]+)-(?<currentValue>[^\s]+)/
```
I think we can make it more strict. runner name can only be a-z.

maybe we need to support different cases too, as some users prefer MacOS. not sure if it's supported by GitHub.

Reply:
What will this do on `macos-13-xl`?
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#choosing-github-hosted-runners